service/src/main.rs

```diff
@@ -107,11 +107,6 @@ fn main() {
 
 	GlobalTokioHandle::initialize();
 
-	#[cfg(feature = "production")]
-	println!("*** Starting service in SGX production mode");
-	#[cfg(not(feature = "production"))]
-	println!("*** Starting service in SGX debug mode");
-
```

Comment:
Having these `println!` here, prints the information regardless of the parameter/mode we're calling the service. Before we had these only for `run`. If I understand correctly, @clangenb 's python script for calling `init-shard` parses the output to get some information (MRENCLAVE?). So having these `println!` here broke the script for that local setup.

Reply:
Oh, that's a good one. Good job finding this one.

Reply:
okay, I've changed it to `info!` log the information, but keep it at the same place (i.e. always log the information, independent of the parameters provided)